src/BloomExe/Edit/WebThumbNailList.cs

```diff
@@ -707,6 +707,11 @@ private void GridReordered(string s)
 						"That change is not allowed. Front matter and back matter pages must remain where they are");
 					var caption = LocalizationManager.GetString("PageList.CantMoveXMatterCaption",
 						"Invalid Move");
+					if (_pages[i].Book.LockedDown)
+					{
+						msg = LocalizationManager.GetString("PageList.CantMoveVernacular",
```

Comment:
"Verncular" isn't the thing here... how about "PageList.CannotReorderWhenTranslating"?